bower.json

```diff
@@ -3,7 +3,8 @@
   "version": "0.0.0",
   "homepage": "https://github.com/kpdecker/six-speed",
   "authors": [
-    "kpdecker <kpdecker@gmail.com>"
+    "kpdecker <kpdecker@gmail.com>",
+    "mmeent <boekewurm@gmail.com>"
```

Comment:
We have a number of other contributors at this point and I'm not sure it's maintainable to list every use in all of these places (particularly when this project isn't published to bower or npm, etc). I think it's better to let github show who has contributed to avoid maintenance issues there. The has the added bonus of also showing what has been contributed, which is more important IMO :)